src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -174,6 +183,43 @@ static long long countEntities
   return c;
 }
 
+/* ****************************************************************************
+*
+* countCmd -
+*
+*/
+static unsigned int countCmd(const std::string& tenant, const BSONArray& pipelineForCount)
+{
+  BSONObj result;
+  BSONObj cmd     = BSON("aggregate" << COL_ENTITIES <<
+                         "cursor" << BSON("batchSize" << BATCH_SIZE) <<
+                         "pipeline"  << pipelineForCount);
+
+  std::string err;
+
+  if (!runCollectionCommand(composeDatabaseName(tenant), cmd, &result, &err))
+  {
+    LM_E(("Runtime Error (ejecuting: %s, error %s)", cmd.toString().c_str(), err.c_str()));
+    return 0;
+  }
+
+  // Processing result to build response
+  LM_T(LmtMongo, ("aggregation result: %s", result.toString().c_str()));
+
+  std::vector<BSONElement> resultsArray = std::vector<BSONElement>();
+
+  if (result.hasField("cursor"))
+  {
+    resultsArray = getFieldF(getObjectFieldF(result, "cursor"), "firstBatch").Array();
+    return getIntFieldF(resultsArray[0].embeddedObject(), "count");
+  }
+  else
+  {
+    LM_E(("Runtime Error (ejecuting: %s, result hasn't cursor field"));
```

Comment:
Spanish typo? 😄

Reply:
Yes :)

Fixed in 97654564ec247e4e4c25e87e5e3f88a36f1690cd